readme.md

```diff
@@ -122,6 +124,13 @@ of this particular output should be. Reference the **Debug Level Chart**
 3. Very Detailed and in-depth Output.
 4. Highly Fine Grained Detailed and in-depth Output.
 
+
+# Database
+### how to connect using `SEQUEL PRO`
+
+You can find more information into the config file `db/config/config.json`
```

Comment:
small typo:

You can find more information _into_ the config file `db/config/config.json`
You can find more information _about_ the config file `db/config/config.json`